Entities/MainObjects/Player.tscn

```diff
@@ -1341,11 +1341,11 @@ stream = ExtResource("58")
 bus = &"SFX"
 
 [node name="RingMagnet" type="Area2D" parent="."]
-visible = false
```

Comment:
explain/revert

Reply:
Ooh yup that shouldn't be visible. Reverted.